src/js/smooch.jsx

```diff
@@ -58,7 +58,7 @@ function renderLink() {
 function getDeviceId() {
     const SK_STORAGE = 'sk_deviceid';
     const deviceId = storage.getItem(SK_STORAGE) ||
-        uuid.v4().replace(/-/g, '');
+    uuid.v4().replace(/-/g, '');
```

Comment:
ok wtf I tinkered with my esformatter version and now vim has busted behavior too.